### DISCUSSION TITLE
🚀 Announcing the launch of the enhanced App validation tool in Developer Portal! 🎉

### DISCUSSION THREAD

**Discussion by surbhigupta12** [MicrosoftDocs/msteams-docs#11459] [Announcements]
### ❓ What does this mean?

If you're an ISV developer, our updated App validation tool lets you evaluate your app packages using the same test cases Microsoft uses for reviews. It identifies errors or warnings in app manifests, basic info, and bot capabilities, providing clear guidelines for resolution. Developers now get detailed success, error, and warning statuses based on new bot capability scenarios, helping them fix issues before submission.

This enhancement aims to provide full visibility, improve app quality at submission, and speed up the publishing process!

![image](https://github.com/user-attachments/assets/2bc9882a-1f5c-4f06-84f6-b83774d7f238)

### 📑 Want to learn more? Here are some resources:

- The [docs](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/manage-your-apps-in-developer-portal#publish&referrer=announcement) give you all the details.

### ❔ Still have queries?

- Feel free to post your feedback here in the comments section.
- For bugs or deeper questions, raise an issue on the [documentation page](https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/manage-your-apps-in-developer-portal#see-also) by selecting open documentation issue.

#### 📢 Subscribe for more updates: 
- [Configure to the RSS feed to get regular platform updates](https://learn.microsoft.com/en-us/microsoftteams/platform/whats-new?pivots=ga-feature#get-latest-updates)

### 💯 A massive thanks to everyone in the community. Your feedback and engagement have been valuable.

**Comment by ragamo** [comment]
The public app validator in https://dev.teams.microsoft.com/appvalidation.htm is broken. Can you fix it please?
